src/app/packages/core/models/overlay.model.ts

```diff
@@ -1,4 +1,12 @@
 import { GeometryObject } from 'geojson';
+import { LimitedArray } from '../utils/limited-array';
+
+export interface OverlaysFetchData extends LimitedArray{
+	data: Overlay[],
+	// number of overlays removed from total overlays (according to config)
+	// when positive, there are more available overlays than what is currently displayed by the UI
```

Comment:
when positive?

Reply:
should I change it to "bigger than zero"?